packages/block-library/src/embed/index.js

```diff
@@ -17,6 +17,8 @@ export const settings = getEmbedBlockSettings( {
 	title: __( 'Embed' ),
 	description: __( 'The Embed block allows you to easily add videos, images, tweets, audio, and other content to your post or page.' ),
 	icon: embedContentIcon,
+	// Unknown embeds should not be responsive by default.
```

Comment:
What effect does this have on existing embeds elsewhere? Are we just always saying they need to be opt-in going forward? That seems like a deprecation/change we should highlight. Maybe I'm misunderstanding 🤷‍♂️

Reply:
No affect at all for the rest of the embed blocks. By setting it to `false` on the generic embed block, we won't apply responsive styles on embeds we don't have a specific block for and end up rendered by the generic embed block. If it's left out, it defaults to true, so the provider specific blocks all get the responsive treatment, except WordPress which opts out.

Reply:
Ah, okay, cool. Thanks, sorry about my confusion!

Reply:
No problem! Embeds are, after all, totally straightforward and there's hardly any code to read and understand. No, hang on, the opposite of that. :laughing: